### ISSUE TITLE
After enabling responsive compression in the asp.net core project, Hangfire’s Dashboard UI cannot be displayed

### ISSUE THREAD

**Issue by berkerdong** [HangfireIO/Hangfire#1980]
After enabling responsive compression in the asp.net core project, after using gzip or br compression, the Dashboard UI of hangfire cannot be displayed, and the screen will be blank.

```
builder.Services.AddResponseCompression(options =>
{
        options.EnableForHttps = true;
});

app.UseResponseCompression();
```

**Comment by odinserj** [comment]
Sorry, I was unable to reproduce this behavior after enabling the compression. I've tried to place `UseResponseCompression` method call before and after `UseHangfireDashboard` and reloaded page with invalidating caches - no luck, dashboard was displaying fine with the following headers (so compression was working).

```
cache-control: no-cache, no-store
content-encoding: br
content-type: text/html
```

**Comment by berkerdong** [comment]
I am using a project with .net 6 mvc architecture, this problem does exist

**Comment by odinserj** [comment]
Could you share some minimal sample repo on GitHub that reproduces the issue?

**Comment by berkerdong** [comment]
I’ll be a bit late and make a warehouse to reproduce the problem

**Comment by berkerdong** [comment]
> Could you share some minimal sample repo on GitHub that reproduces the issue?

[https://github.com/berkerdong/HangfireDemo.git](https://github.com/berkerdong/HangfireDemo.git)   

This is a HangfireDemo I built. I made a page for the login interface. It works normally when gzip or br responsive compression is turned off. The main interface of hangfire cannot be loaded after responsive compression is turned on.

Note that it can run normally after the following line

app.UseResponseCompression();
